Packs/CommonPlaybooks/Playbooks/playbook-IP_Enrichment_-_Internal_-_Generic_v2.yml

```diff
@@ -482,12 +541,27 @@ inputs:
     Whether to convert the IP address to a hostname using a DNS query (True/False).
     The default value is true.
   playbookInputQuery:
+- key: ExecutedFromParent
+  value:
+    simple: "False"
+  required: false
+  description: |-
+    Whether to execute common logic, like the classification of IP addresses to ranges and resolving, in the the main (IP Enrichment - Generic v2) enrichment playbook, instead of in the sub-playbooks.
```

Comment:
Same as the parent comment - let's make it shorted + Possible values + default value.

Reply:
Done